components/magic.js

```diff
@@ -0,0 +1,32 @@
+async function cardCmd(message, args)
+{
+    searchCard(args[0])
+    .then((card) => {message.channel.send(card.image);})
```

Comment:
You should add a `.catch((err) => ())` at the end to handle errors (for when promise calls reject() instead of resolve())

Reply:
You still need a resolve here to catch the "rejects" thrown by `searchCard()`, should look like
`searchCard(args[0])
    .then(() => ...)
    .catch((error) => send somthing to discord to let the user know it failed)`